src/main/java/pl/gov/coi/pomocua/ads/materialaid/domain/dto/MaterialAidOfferDto.java

```diff
@@ -0,0 +1,31 @@
+package pl.gov.coi.pomocua.ads.materialaid.domain.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import pl.gov.coi.pomocua.ads.dictionaries.domain.City;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+
+@Setter
+@Getter
+@Builder
+public class MaterialAidOfferDto implements Serializable {
```

Comment:
We've introduced a `BaseOffer` entity that has id, title and description. I recommend you extend from it.

Reply:
Ok. It hasn't been implemented yet when I pulled the repo.